js/admin.block-post.js

```diff
@@ -352,7 +352,7 @@
 			const postType = $( input ).val();
 			if ( -1 === excludedPostTypes.indexOf( postType ) ) {
 				displayList.push(
-					$( input ).next( 'label' ).text()
+					$( input ).next( 'label' ).text(),
```

Comment:
I should probably revert this, as this file isn't compiled. And [support](https://caniuse.com/#feat=mdn-javascript_grammar_trailing_commas_trailing_commas_in_functions) for trailing commas in functions doesn't look good enough.